airbyte-integrations/connectors/source-plausible/source_plausible/spec.yaml

```diff
@@ -36,4 +36,4 @@ connectionSpecification:
         Start date for data to retrieve, in ISO-8601 format.
       pattern: ^[0-9]{4}-[0-9]{2}-[0-9]{2}$
       examples:
-        - YYYY-MM-DD
+        - YYYY-MM-DD
```

Comment:
Please remove the file from this PR as it is unrelated to the LinkedIn Ads source connector.